mungegithub/submit-queue/.gitignore

```diff
@@ -1,3 +1,5 @@
 local.deployment.yaml
 local.secret.yaml
 local.service.yaml
+local.pvc.yaml
+local.pv.yaml
```

Comment:
I think its actually safer to use the `local.*.yaml` version. If a secret, service, or persistent volume template is added to an app this will prevent a new local.XXX.yaml file from being committed without requiring an update to .gitignore. Any files that are intended to be committed wont match that pattern.